kratos/geometries/geometry.h

```diff
@@ -1848,6 +1848,63 @@ class Geometry
         return rResult;
     }
 
+    /**
+     * @brief Returns the local coordinates of a given arbitrary point, considering additionally
+     *              a certain increment in the coordinates
+     * @param rResult The vector containing the local coordinates of the point
+     * @param rPoint The point in global coordinates
+     * @return The vector containing the local coordinates of the point
+     */
+    virtual CoordinatesArrayType& PointLocalCoordinates(
```

Comment:
@KratosMultiphysics/technical-committee do we want to add this extra method? 

personally i don't love the idea of overloading once again the (already bloated) geometry interface, however i honetly don't see a way to do this operation otherwise (other than having an auxiliary geometry and copying the node positions to that)

@vryy on my side this addition is the only thing that could take some time to be approved. I would consdier the const additions as a FastPR, but not this one

Reply:
@RiccardoRossi I think this is mainly to improve the consistency of the geometry where functions supporting for DeltaPosition is added. Probably you are right, it looks more complicated now, but I think it should be there to be complete, at least for the case to compute local projection in the reference configuration.